bindings/ios/MEGABackgroundMediaUpload.h

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file MEGABackgroundMediaUpload.h
+ * @brief Time zone details
```

Comment:
I think this is not the brief of this class.

Reply:
changed to "Background media upload". good catch, thanks